src/storage/pubsub/topic.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "index.h"
+
+#include <cc_bstring.h>
+
+/*
+ * a topic is an endpoint that clients can subscribe to, equivalent to
+ * "channel" in the original redis protocol.
```

Comment:
Oh yea I should explain. mostly ccommon (which Pelikan depends on and includes) already have the concept of a "channel" to mean some kind of connection (tcp, uds, pipe...)